project1/css/index.css

```diff
@@ -62,3 +64,101 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+nav {
+    flex-grow: 1;
+}
+header{
+    display: flex;
+    justify-content: flex-end;
+    flex-wrap: wrap;
+}
+header img{
+    width:150px;
+    padding-top: 20px;
+    padding-bottom: 80px;
+    padding-right: 10%
+    flex-shrink;
```

Comment:
Hmm, I think you called flex-shrink incorrectly here. And I don't think it is necessary here :)